sdk/eventhub/Azure.ResourceManager.EventHubs/CHANGELOG.md

```diff
@@ -47,8 +47,8 @@ Example: Create an Event Hub:
 
 Before upgrade:
 ```csharp
-using Microsoft.Azure.Management.EventHub;
-using Microsoft.Azure.Management.EventHub.Models;
+// using Microsoft.Azure.Management.EventHub;
+// using Microsoft.Azure.Management.EventHub.Models;
```

Comment:
I recommend having a separate snippet block for imports to avoid having to do this in the source where it's not possible outside of comments. I made a similar comment on the Service Hub PR.

Reply:
I'm not a fan of the separate block, personally.  I think it harms locality and context.   That said, if we've agreed on that as a common format, I'll conform.